src/biome/text/pipeline.py

```diff
@@ -286,17 +288,24 @@ def predict(self, *args, **kwargs) -> Dict[str, numpy.ndarray]:
         """
         return self._model.predict(*args, **kwargs)
 
-    def explain(self, *args, **kwargs) -> Dict[str, Any]:
+    def explain(self, *args, n_steps: int = 5, **kwargs) -> Dict[str, Any]:
         """Predicts over some input data with current state of the model and provides explanations of token importance.
 
         The accepted input is dynamically calculated from head.input
 
+        Parameters
+        ----------
+
+        n_steps: int
+            The number of steps for token attribution calculation (if proceed).
+            If the number of steps is less than 1, the attributions will not be calculated
+
```

Comment:
I think it should be possible to dynamically copy the docstring from the corresponding `TaskHead.explain_prediction()` or `_model.explain()` method. It is a bit strange that `n_steps` is the only parameter documented here, and the rest goes into the args and kwargs.

Reply:
mmmm the explain/predict methods have dynamic arguments calculated directly from head.inputs. Normally these inputs correspond to `Head.featurize` method, but not always (the `RecordClassification` head set the pipeline input names in configuration.